web/src/settings.ts

```diff
@@ -114,6 +114,8 @@ export function build_page(): void {
         twenty_four_hour_time_values: settings_config.twenty_four_hour_time_values,
         general_settings: settings_config.all_notifications(user_settings).general_settings,
         notification_settings: settings_config.all_notifications(user_settings).settings,
+        customize_stream_notifications_options:
```

Comment:
I think `stream_specific_notification_settings` is a better variable name since the notification settings are same, we are just providing a widget here to customize them for a stream.